Decompression/GZipCompressor.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include "CommonMeshRepresentation.h"
+#include "MeshImporter.h"
+#include <string>
+#include <stdio.h>
+#include <stdlib.h>
+#include <fstream>
+#include <iostream>
+#include <boost/iostreams/filtering_streambuf.hpp>
+#include <boost/iostreams/copy.hpp>
+#include <boost/iostreams/filter/gzip.hpp>
+
+using namespace std;
```

Comment:
Do you really need to use __whole__ namespace std?
Did you consider writing something like:
use std::what_i_need